website/source/docs/guides/index.html.md

```diff
@@ -10,9 +10,9 @@ description: |-
 
 # Consul Guides
 
-~> The Consul guides have moved to the [HashiCorp Learn platform](https://learn.hashicorp.com/consul).
+~> The Consul guides have moved to the [HashiCorp Learn platform](https://learn.hashicorp.com/consul?utm_source=consul.io&utm_content=guide-index).
```

Comment:
should we do `utm_medium=docs` to be consistent with the changes you made on monday?